homeassistant/components/group/__init__.py

```diff
@@ -257,7 +244,7 @@ def groups_with_entity(hass: HomeAssistantType, entity_id: str) -> List[str]:
 
             extra_arg = {
                 attr: service.data[attr]
-                for attr in (ATTR_VISIBLE, ATTR_ICON, ATTR_VIEW, ATTR_CONTROL)
+                for attr in (ATTR_ICON)
```

Comment:
Now you're iterating over each character in the string, one comma is all the difference…

```suggestion
                for attr in (ATTR_ICON,)
```

Reply:
Are we on top of this, to fix it?

Reply:
Sorry, seeing the notification just now. Adjusting.

Reply:
https://github.com/home-assistant/home-assistant/pull/32032

Reply:
Not sure why I merged this without being addressed 🤦‍♂

Reply:
Before coffee ☕️  😉 

All good now 👍